net-dialup/freeradius-client/freeradius-client-1.1.7-r1.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools
+
+DESCRIPTION="FreeRADIUS Client framework"
+HOMEPAGE="https://wiki.freeradius.org/project/Radiusclient"
+SRC_URI="ftp://ftp.freeradius.org/pub/freeradius/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86"
+
+IUSE="scp shadow static-libs"
+
+DEPEND="!net-dialup/radiusclient-ng"
+RDEPEND="${DEPEND}"
+
+DOCS="BUGS doc/ChangeLog doc/login.example doc/release-method.txt
```

Comment:
You can match the `*` to represent a real existing file extension. There can't be many READMEs right?

Make it back to an array, portage will convert this string to an array anyway.